src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1035,6 +1035,23 @@ void createEntityWithDiskBaseModelAndWriteOut() {
             assertNotNull(rtest, "Entity does not exist");
         }
 
+        /**
+         * A cluster should be able to contain a link to one or more external KBs
+         */
+        @Test
+        void createClusterWithLinkAndConfidence() {
+            // create a valid entity and cluster
+            Resource typeAssertionJustification = makeTextJustification(model, "NYT_ENG_20181231",
+                    42, 143, system, 0.973);
+            Resource newEntity = makeEntity(model, getEntityUri(), system);
+            markJustification(addType(newEntity, SeedlingOntology.Person), typeAssertionJustification);
```

Comment:
It looks like this code was borrowed from the NIST examples (setup?) where we use `addType()` and justify all type assertions.  Although it's not bad or wrong to do any of this, it is a bit non-parallel with other non-NIST AIF tests, and it will look more so once we break this file up into AIF, NIST, and NIST-hypothesis in AIDA-719.  This will also mean the Example on the Confluence page (if we add it) will be streamlined.

So I'd suggest changing the example a bit to be more like the other AIF examples above it.  Take a look at `createASimpleCluster()` where it creates an entity, marks its type, and eventually adds it to a cluster.  In fact, one could argue that the code could be the same as `createASimpleCluster()`, but with the `linkToExternalKB()` call added.